### ISSUE TITLE
Deprecation warning win-spawn

### ISSUE THREAD

**Issue by nerdbeere** [james-proxy/james#63]
I get the following deprecation warning on `npm install`:

```
win-spawn@2.0.0: use [cross-spawn](https://github.com/IndigoUnited/node-cross-spawn) or [cross-spawn-async](https://github.com/IndigoUnited/node-cross-spawn-async) instead.
```

**Comment by mitchhentges** [comment]
Yes, it's an issue with one of our (up-to-date) dependencies: `electron-connect`. It uses `win-spawn`.
I'll open a PR for that project

**Comment by mitchhentges** [comment]
Should be fixed, `electron-connect` just released `0.3.6`
